### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by ebenezer-melkamu**
### Modifications 
1. Updated the name of the package from `ra-soul-provider` to `ra-soul` 
2. Added a note on how to use the `extension` feature of soul with `RA` app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,3 +106,7 @@ function App() {
   );
 }
 ```
+
+## Running a Compiled React Admin App with Soul
+
```
Comment (IanMayo) [inline README.md:111]: @AbegaM - didn't we have to create a custom route to serve the static content?

image

**Review by IanMayo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IanMayo) [reply README.md:111]: (I did provide that feedback earlier this afternoon @AbegaM - but I clicked `review` instead of `submit comment`

Comment (ebenezer-melkamu) [reply README.md:111]: Hello @IanMayo, This is the `RA-Provider`. Shouldn't we point the extension example to the `Soul` repository, considering that the extension feature is related to `Soul`? So should we add the above code example to the `Soul` repo?


Screen Shot 2024-06-14 at 10 07 10 AM

Comment (IanMayo) [reply README.md:111]: Ooh, that's a difficult one. Extensions are intended for consumers of Soul to use, and RA-Soul is a consumer.  But, it's such a valuable extension that yes - I think it is worth documenting it in Soul, with some more explanation.

Comment (IanMayo) [reply README.md:111]: Thumbs-up from Vahid :-D
 
So, maybe:
- [ ] document how to host static content in `Soul`, in `Soul` repo
- [ ] update this PR's link to that content

Comment (ebenezer-melkamu) [reply README.md:111]: Ok Thanks, i will push a PR to `Soul` and will also update the `README` file in this repo

**Review by IanMayo — APPROVED** [review]

**Pull request merged**